src/sentry/api/endpoints/user_notification_details.py

```diff
@@ -58,13 +29,12 @@ def serialize(self, obj, attrs, user, *args, **kwargs):
         raw_data = {option.key: option.value for option in attrs}
 
         data = {}
-        for key in USER_OPTION_SETTINGS:
-            uo = USER_OPTION_SETTINGS[key]
+        for key, uo in USER_OPTION_SETTINGS.items():
             val = raw_data.get(uo["key"], uo["default"])
             if uo["type"] == bool:
-                data[key] = bool(int(val))  # '1' is true, '0' is false
+                data[key.value] = bool(int(val))  # '1' is true, '0' is false
```

Comment:
why is this now `key.value` instead of `key`?

Reply:
I updated `USER_OPTION_SETTINGS` from mapping strings to objects to mapping enums to objects.